### PR TITLE
feat: build using golang version 1.17.13

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: Title
Ticket: MEN-5754

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --pr mender-connect/77

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ stages:
   - trigger
 
 variables:
-  GOLANG_VERSION: "1.17.6"
+  GOLANG_VERSION: "1.17.13"
```
Comment (merlin-northern) [inline .gitlab-ci.yml:8]: can we take this opportunity to verify what we can bump the version of go in the acceptance tests? last time I checked it was 1.12 what version does kirkstone have?

Comment (kacf) [reply .gitlab-ci.yml:8]: > can we take this opportunity to verify what we can bump the version of go in the acceptance tests?

We cannot bump the version there, we are at the mercy of whatever Yocto uses. If it was 1.12, it probably still is. Any vulnerabilities will have to be handled by the Yocto people (but we can report it, of course).

**Review by kacf — APPROVED** [review]

**Review by lluiscampos — COMMENTED** [review]
This version bump (here) only affects unit tests so it shouldn't have a changelog entry. But as we don't have release notes for `mender-dist-packages` I don't really know where we should place such changelog entry :thinking:

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Requesting changes so that we can further discuss the changelog entry detail. I think we should remove it from here.

**Comment by lluiscampos** [comment]
> This version bump (here) only affects unit tests so it shouldn't have a changelog entry. But as we don't have release notes for `mender-dist-packages` I don't really know where we should place such changelog entry thinking

I was wrong there. The `GOLANG_VERSION` is indeed used by `mender-dist-packages` to determine which version to use. However my concern wrt the changelog entry still stands.

**Comment by tranchitella** [comment]
@lluiscampos I removed the changelog entry.
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Review by lluiscampos — APPROVED** [review]

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --pr mender-connect/77 --pr mender-qa/599

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --pr mender-connect/77

**Pull request merged**